### PR TITLE
[SG-133] upgrading to sdk v0.47

### PR DESCRIPTION
**Pull request by spoo-bar**
- Replacing tendermint with cometbft
- Updating sdk to 0.47 
- Updating wasmd to 0.41 
- Updating ibc-go to v7
- Removing REST stuff
- Replacing ibc-hooks fork with reference to [github.com/cosmos/ibc-apps/modules/ibc-hooks/v7](https://github.com/cosmos/ibc-apps/tree/main/modules/ibc-hooks)
- All the existing gov props for x/cron and x/globalfee are kept as legacy gov props. Will update them in later PR
- Fixing simapp references in tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -157,7 +157,7 @@ DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace $(
 ###############################################################################
 ###                                Protobuf                                 ###
 ###############################################################################
-PROTO_BUILDER_IMAGE=ghcr.io/cosmos/proto-builder:0.9.0
+PROTO_BUILDER_IMAGE=ghcr.io/cosmos/proto-builder:0.14.0
```
Comment (jhernandezb) [inline Makefile:160]: seems like the recommended version was 0.11.2 is there any difference with this version?

File: cmd/starsd/cmd/genesis_accounts.go [hunk 2]
```diff
@@ -1,193 +1,195 @@
 package cmd
 
-import (
```
Comment (jhernandezb) [inline cmd/starsd/cmd/genesis_accounts.go:3]: If this is already supported we can remove this file

File: internal/statesync/snapshotter.go [hunk 3]
```diff
@@ -3,13 +3,12 @@ package statesync
 import (
```
Comment (jhernandezb) [inline internal/statesync/snapshotter.go:3]: we can remove this custom snapshotter if statesync works in v0.47 just need to check

File: Makefile (hunk 1 above)
Comment (spoo-bar) [reply Makefile:160]: Currently recommend version is > 0.11.5 for sdk 47.
Using 0.14.0 just to use the most upto date builder.
The latest image has some optimizations as well https://github.com/cosmos/cosmos-sdk/pull/16343

**Pull request merged**